README.md

```diff
@@ -110,4 +110,30 @@ A script will run automatically, asking for your settings and desired configurat
 
 <hr>
 
-**MeiliSearch** provides and maintains many **SDKs and Integration tools** like this one. We want to provide everyone with an **amazing search experience for any kind of project**. If you want to contribute, make suggestions, or just know what's going on right now, visit us in the [integration-guides](https://github.com/meilisearch/integration-guides) repository.
\ No newline at end of file
+**MeiliSearch** provides and maintains many **SDKs and Integration tools** like this one. We want to provide everyone with an **amazing search experience for any kind of project**. If you want to contribute, make suggestions, or just know what's going on right now, visit us in the [integration-guides](https://github.com/meilisearch/integration-guides) repository.
+
+
+# How to update MeiliSearch image on Marketplace on new Release:
+
+1. Update version number `vX.X.X` on `scripts/deploy.sh`
+
+  * Update the `wget` command for downloading the right Debian release
+  * Update the `git checkout` command so the image is built from the right git tag.
+
+2. Update version number `vX.X.X` on `server/tools/build-image.py`
+
+  * Update the `MEILI_VERSION_TAG` constant.
+
+3. Commit your changes on a new branch on git.
+
+4. Create a git tag with the appropriate version number and push to repository `vX.X.X` (in the current working branch)
```

Comment:
Not sure to really understand. Do you mean pushing the tag on the repo with `git push --tag origin <your-branch>`?